thirdparty/minizip/unzip.c

```diff
@@ -68,10 +68,6 @@
 #include <stdlib.h>
 #include <string.h>
 
-#ifndef NOUNCRYPT
-        #define NOUNCRYPT
-#endif
-
```

Comment:
This change felt really strange: Why was `NOUNCRYPT` added(?), yet the matching `NOCRYPT` is nowhere to be found. Without this change it's possible to encrypt files, but not decrypt them. If there was a specific reason for the addition/removal, it should probably be commented or maybe the definition be added to the Scons files.